mjpc/estimators/trajectory.h

```diff
@@ -33,7 +33,6 @@ class EstimatorTrajectory {
   EstimatorTrajectory() { Initialize(0, 0); }
   EstimatorTrajectory(int dim, int length) { Initialize(dim, length); }
 
-
   // initialize
   void Initialize(int dim, int length) {
```

Comment:
Maybe not for this PR, but do you need an Initialize method? Can you use the constructor instead, and possibly offer move/copy constructors to overwrite an existing EstimatorTrajectory?

Reply:
We can change the interface. This will require some changes to classes that have `EstimatorTrajectory` members.